### ISSUE TITLE
Network unavailable;sinal strength-bars icon shows full signal

### ISSUE THREAD

**Issue by krustbr** [mmontuori/atrix-mrom#7]
As i said on XDA, the network gone, only "emergency calls" available, but the sinal strength icon shows "full signal" on notification bar instead of a no signal icon.
Ive started logcat, with the phone on this situation, then ive made these steps:
Tried to make a call - network unavailable
Switch the airplane mode ON/OFF to try to rejoin the network
Tried to call again - network unavailable
Tried to search available networks

Here the log of these steps: https://rapidshare.com/files/4036671549/androiderrornetwork.txt

The only fix i found so far to get back the network is to restart the phone.

**Comment by mmontuori** [comment]
Will investigate soon...

**Comment by mmontuori** [comment]
I looked at logs and saw the following error being repeated:

I/TelephonyRegistry( 1489): notifyServiceState: 1 home BRA SCTL SCTL 72415 (manual) EDGE CSS not supported -1 -1RoamInd: -1DefRoamInd: -1EmergOnly: true

Could this be related to APN settings?

**Comment by krustbr** [comment]
Dont know, i didnt change my apn settings. The weird thing is that when the error occurs i did a search for available networks and the both networks 3g one and the 2g are available. Anything i can do to search deeper through the APN settings? 
.update: 
the problem appears again, this BRA SCTL SCTL network appears on "status" on the configs, but it isnt a known network here from brazil, dont know what it is, and i cant search the availabe networks, i took a screenshot from the about: http://imgur.com/ia9HJ

When i try to "search" the available networks operators it cant find no one: http://imgur.com/nELxD

**Comment by mmontuori** [comment]
This is definitely an APN issue, search for your carrier APN settings and apply them to the phone manually, if you give them to me, I can include them in the next version.

The search takes a long time sometimes, let it finish completely...

**Comment by krustbr** [comment]
I try to add the carrier APN settings, but, i create a new apn, insert the data, when i save has only ONE apn listed(mms apn) , i cant add a new one, it isnt listed.

The issue of network unavailable persists, when i perform a network search , it find the network listed as unavailable but i cant join/register at this network.

I compiled the APN settings of brazilian carriers:  http://pastebin.com/xM5TX1Xk wish it helps!

**Comment by mmontuori** [comment]
Time permitting I will create a flashable file containing the APNs that you could flash on top of an existing build...that should take care of it...

**Comment by mmontuori** [comment]
Is this still a problem?

**Comment by krustbr** [comment]
The issue has decreased well, but sometimes still happen. Ive tried to do some APN reconfigs, but with no success.
I guess, when the signal has lost the phone isnt reconnecting on the same or another network from the same operator.

**Comment by mmontuori** [comment]
Looks like APNs are not the problem here, the APN listed in your file are already present in the build... I was going to look at the logcat some more, but it's not there anymore, do you still have it?

**Comment by krustbr** [comment]
Yep, still have, should i send to an email?Its too big to paste on pastebin. Ive noted that my carrier has 2 networks 2G and 3G, when im connected to 3G and i lost signal - emergency calls only - i need to perform a new network scan, choose the same 3G or change to 2G from the same carrier and im back to network, he didnt perform these steps by itself. 
- Wich what e-mail should i send the log?!

**Comment by mmontuori** [comment]
you can use michael.montuori@gmail.com to send logs...

**Comment by bulutgz** [comment]
i havent got any signal bars showing cross
